src/Type/ObjectType/RootQuery.php

```diff
@@ -220,6 +221,12 @@ public static function register_type() {
 
 							return absint( $post_id ) ? $context->get_loader( 'post' )->load_deferred( $post_id )->then(
 								function ( $post ) use ( $allowed_post_types ) {
+
+									// if the post isn't an instance of a WP_Post or Post model, return
+									if ( ! $post instanceof \WP_Post && ! $post instanceof Post ) {
```

Comment:
Should we be converting a `WP_Post` to a `Post` model before returning on L234?
(Might be beyond the scope of this PR, but if we're explicitly allowing `\WP_Post` and not just its `Model` on the loader, it might make sense to address it now).

Reply:
ah! Yes, this is actually already a Post model, not a WP_Post. I thought I had changed it before I committed last night, but I apparently did not. Good catch! 🙌🏻